### PR TITLE
Royalty Changes

### PR DESCRIPTION
**Pull request by TheEighthDeadlySin**
Created this pull request in order to change the functionality of the royalty, with more roles that can become the next King. Reminder, the following below is the order of priority at which the roles will replace the King:

King <-- Prince <-- Queen <-- Princess

### CODE REVIEW

**Review by Topkinsme — COMMENTED** [review]
Looks good, Queen just needs a power rating.

File: royalty/queen [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Queen - ()
+- Takes the place of the King, if they are alive and there is no alive Prince when the King dies.
+- Upon death, if the cause of death is anything other than tribute, the Chief Warrior of their team will gain and extra attack the following night.
```
Comment (12Metatron) [inline royalty/queen:3]: type here, "will gain and extra attack"

Comment (12Metatron) [reply royalty/queen:3]: fix the typo and make the first line phrased better. something like
"If the King dies, a living Queen will become the new King. Prince takes priority over Queen if both are alive."

**Review by 12Metatron — CHANGES_REQUESTED** [review]

File: royalty/princess [hunk 2]
```diff
@@ -0,0 +1,6 @@
+Princess- (4)
+- Takes the place of the King, if they are alive and there is no alive Prince or Queen when the King dies.
+- The first time the Princess is about to die to tribute, the Princess will not die, however will be revealed to everyone. This ability is then lost.
+- The Princess will receive all of the money paid towards tribute when the passive ability is activated.
```
Comment (12Metatron) [inline royalty/princess:4]: should clarify, the combined money paid by all teams

File: royalty/queen (hunk 1 above)
Comment (12Metatron) [reply royalty/queen:3]: also needs a power rating before i can merge

Comment (TheEighthDeadlySin) [reply royalty/queen]: One question, what do you think the power level of Queen should be? It seems relatively powerful, however the ability only activates upon death to anything other than tribute, which of course, has the risk of the King dying in that time, causing the Queen to also die if no Prince is present.

Comment (TheEighthDeadlySin) [reply royalty/queen]: Also, the ability is basically null if the Chief Warrior is respawning during that time.

File: royalty/princess (hunk 2 above)
Comment (Topkinsme) [reply royalty/princess]: I think this is resolved

File: royalty/queen (hunk 1 above)
Comment (Topkinsme) [reply royalty/queen]: @12Metatron Please do let us know what power level the Queen should have

Comment (Topkinsme) [reply royalty/queen]: Looks good to me

File: royalty/queen [hunk 3]
```diff
@@ -0,0 +1,4 @@
+Queen - (7)
+- If the King dies, a living Queen will become the new King. Prince takes priority over Queen if both are alive.
+- Upon death, if the cause of death is anything other than tribute, the Chief Warrior of their team will gain an extra attack the following night.
```
Comment (Topkinsme) [inline royalty/queen:3]: Actually I have a question, is this power a 1-time thing?

Comment (TheEighthDeadlySin) [reply royalty/queen:3]: No, it is every time the Queen dies to anything other than tribute.

Comment (Topkinsme) [reply royalty/queen:3]: Alright

**Review by Topkinsme — APPROVED** [review]
Looks good

**Pull request merged**